### ISSUE TITLE
Sidebar issues

### ISSUE THREAD

**Issue by jschirrmacher** [mrsimpson/energy#27]
After the last changes, at first, I didn't recognized that there are more input fields in a kind of sidebar on the left. This sidebar is hidden by default, presumably b/c it will use less space on mobile devices. On a notebook (even on the 13" I currently use) this is not really necessary. Instead, the layout seems a little bit funny due to the fact that a comparatively narrow white content is placed more or less on the left side of the screen. The button to show the side bar is comparatively small (though, it is extends to the left, so I _should_ have seen it at first, I agree).

<img width="1438" alt="Bildschirmfoto 2022-11-15 um 06 13 17" src="https://user-images.githubusercontent.com/2821879/201831955-855c20ae-e1f7-4683-b862-20a8d26db907.png">

So, how about making the side bar visible by default on larger screens?

And, how about using tabs instead of a side bar on smaller devices?

**Comment by jschirrmacher** [comment]
Another problem in this context: On a small screen (or when using the dev tools on a slightly smaller than my 13"), when the side bar is opened, the main part of the page is shrinked to a very narrow bar, and the text is wrapped, so it makes it nearly unreadable and it moves the handle to close the side bar far down.
<img width="697" alt="Bildschirmfoto 2022-11-15 um 07 06 25" src="https://user-images.githubusercontent.com/2821879/201839102-a02e8e19-e7a9-4bd9-9c4b-f562d0e6b878.png">

**Comment by mrsimpson** [comment]
Agreed, also seen that when experimenting with the layout. Wdyt about moving the parameter below the consumption inputs?

There could be a text near the consumption input which has a link to show them (hidden by default)?

**Comment by mrsimpson** [comment]
Something along the lines of 
```
Basierend auf Deinen Verbrauchsdaten und Deiner angepeilten Einsparung berechnen wir, was das für Deine Gasrechnung bedeutet. 
Dabei unterstellen wir einige <a>Annahmen</a>, die Du <a>hier</a> anpassen kannst" (Same link twice)
```

**Comment by jschirrmacher** [comment]
This is resolved by #33
